lib/src/generic/aes_generic.c

```diff
@@ -330,3 +342,5 @@ void aes_generic_gen_tables(void)
         FT3[i] = ROTL8(FT2[i]);
```

Comment:
Yes, indeed, that's possible, and was almost supported by the code with minor changes.
I added the feature with a compile flag to address this case, it can work in combination with the flag for enabling `static const` LUT.